tests/Output/UnifiedDiffOutputBuilderDataProvider.php

```diff
@@ -0,0 +1,400 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of sebastian/diff.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SebastianBergmann\Diff\Output;
+
+/**
+ * @coversNothing
```

Comment:
?
it is not test class, should not have test-related annotatoin

Reply:
fixed